apis/python/src/tiledb/vector_search/flat_index.py

```diff
@@ -36,12 +36,15 @@ def __init__(
             ctx=self.ctx,
             config=config,
         )
-        self.ids_uri = self.group[
-            storage_formats[self.storage_version]["IDS_ARRAY_NAME"] + self.index_version
-        ].uri
-        if tiledb.array_exists(self.ids_uri, self.ctx):
-            self._ids = read_vector_u64(self.ctx, self.ids_uri, 0, 0)
-        else:
+        try:
```

Comment:
Do you mind to add some comment inline (so it doesn't have to be reconstructed from git-blame):
- about why we are doing this and what version/pre-version we need this for
- TODO that we should deprecate and remove support for the older version(s)

Reply:
Also is there a reason for the try block instead of checking for the group member?

Reply:
I couldn't find a method that checks for group member existence without throwing an exception. Is there such a method?

Reply:
Ok we can use standard contains operation for this. Applied

Reply:
Added a comment about the check and this being ok to not have ids provided. For backwards compatibility, I don't think we need to stop supporting previous versions atm.